pkg/providers/snow/reconciler/reconciler.go

```diff
@@ -156,3 +159,25 @@ func toClientControlPlane(cp *snow.ControlPlane) *clusters.ControlPlane {
 		Other:                       other,
 	}
 }
+
+func toClientWorkers(workers *snow.Workers) *clusters.Workers {
+	other := make([]client.Object, 0, len(workers.CAPASIPPools))
+	for _, p := range workers.CAPASIPPools {
+		other = append(other, p)
+	}
```

Comment:
can we add ip pools to a test? we don't need a new test, just add eks-a ip pools to one of the test's clusters input. And then just verify that the aws snow ip pools get created.

Reply:
updated reconciler test